play/android-publisher/src/main/kotlin/com/github/triplet/gradle/androidpublisher/EditManager.kt

```diff
@@ -67,26 +67,13 @@ interface EditManager {
             retainableArtifacts: List<Long>?
     )
 
-    /** Uploads and publishes the given [bundleFile]. */
+    /** Uploads the given [bundleFile]. */
     fun uploadBundle(
             bundleFile: File,
-            strategy: ResolutionStrategy,
-            didPreviousBuildSkipCommit: Boolean,
-            trackName: String,
-            releaseStatus: ReleaseStatus?,
-            releaseName: String?,
-            releaseNotes: Map</* locale= */String, /* text= */String?>?,
-            userFraction: Double?,
-            updatePriority: Int?,
-            retainableArtifacts: List<Long>?
-    )
+            strategy: ResolutionStrategy
+    ) : Long?
```

Comment:
`uploadBundle` method does not publish the release anymore. Now it works similar to `uploadApk` - it just uploads the file.